src/main/java/org/shanerx/tradeshop/commands/Executor.java

```diff
@@ -544,7 +546,7 @@ public boolean onCommand(CommandSender sender, Command cmd, String label, String
 				sender.sendMessage(colorize("&a" + name + " has been removed from the custom items."));
 				return true;
 
-			} else if (args[0].equalsIgnoreCase("getCI")) {
+			} else if (args[0].equalsIgnoreCase("getCI") || args[0].equalsIgnoreCase("items")) {
```

Comment:
This command should get a single custom item not a list of items, also i think were going to retire custom items for now